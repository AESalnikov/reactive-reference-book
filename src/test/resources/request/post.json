{
  "firstName": "Сергей",
  "lastName": "Сергеев",
  "middleName": "Сергеевич",
  "positionHeld": "Водитель",
  "email": "sergeev-ss@test.ru",
  "phoneNumber": "88000000003"
}
