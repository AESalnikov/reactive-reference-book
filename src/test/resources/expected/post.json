{
  "id": "00000000-0000-0000-0000-000000000003",
  "firstName": "Сергей",
  "lastName": "Сергеев",
  "middleName": "Сергеевич",
  "positionHeld": "Водитель",
  "email": "sergeev-ss@test.ru",
  "phoneNumber": "88000000003"
}